### ISSUE TITLE
scatter disobeys my command (xlim)

### ISSUE THREAD

**Issue by mauro3** [tbreloff/Plots.jl#502]
Scatter doesn't let me freely choose xlimits (pyplot backend):
```
julia> p = scatter(1:5, xlims=(2.0,5))

julia> xlims(p)
(1.91,5.09)

julia> xlims!(p, (2,5))

julia> xlims(p)
(1.91,5.09)
```
I can see the use of choosing them by default such that the points are on-screen.  But I should be able to override this.

**Comment by tbreloff** [comment]
No that sounds like a bug. What commit are you on?

**Comment by mauro3** [comment]
```
commit 90d054c2ec72baef7ce180ef75122973b8dd5cfe
Author: Thomas Breloff <tom@breloff.com>
Date:   2016-09-20 12:05:16 -0400

    glvisualize fixes; limit ticks to amin/amax closes #495
```

**Comment by tbreloff** [comment]
Fixed on master/dev
